include/fmt/format-inl.h

```diff
@@ -522,6 +522,12 @@ FMT_CONSTEXPR void to_esc(uint8_t c, char out[], int offset) {
   out[offset + 1] = static_cast<char>('0' + c / 10 % 10);
   out[offset + 2] = static_cast<char>('0' + c % 10);
 }
+
+FMT_CONSTEXPR void to_esc(uint8_t c, wchar_t out[], int offset) {
```

Comment:
This can be merged into `to_esc` overload above by making the latter a template.